RELEASE.md

```diff
@@ -0,0 +1,11 @@
+# Release
+
+## Instructions for Redeploying Try PureScript
+
+After making a new compiler release, do the following to redeploy Pursuit using the new compiler.
```

Comment:
```suggestion
After making a new compiler release, do the following to redeploy Try PureScript using the new compiler.
```

Reply:
Also, if there are breaking changes then we need to update the package set as described in the main README file.

Reply:
Fixed the name change. I've also ported the package set docs and cleaned them up a bit. We're due for an upgrade as its still on a 0.14.3 package set.